1-sum-app/sum.html

```diff
@@ -0,0 +1,14 @@
+<html>
+    <head>
+        <title>1-sum-app</title>
+    </head>
+        <body>
+           <div class="wrapper-content">
+            <br>
+                <u><h1>Add Two Numbers </h1></u><br>
+                <label><b>First number: </b></label><input type="number" min="0"><br><br>
```

Comment:
Please remove min="0". If we do this we will not be able to enter negative number. This will prevent us from doing next user story.

Reply:
ok sir.